src/util/web_worker_transfer.js

```diff
@@ -119,7 +119,7 @@ function isArrayBuffer(val: any): boolean {
  *
  * @private
  */
-export function serialize(input: mixed, transferables?: Array<Transferable>): Serialized {
+export function serialize(input: mixed, transferables: ?Array<Transferable>): Serialized {
```

Comment:
I'm surprised these aren't equivalent. What's the difference? An optional parameter vs a parameter with an optional value ...

Reply:
[An optional parameter accepts `undefined | T`, while an optional type accepts  `undefined | null | T`.](https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVUCuA7AxgFwEs5swo44AKADwC4wB+bTAWwCMBTAJwEp6A3OIQAmYAN6oAvunJUeAbgwVKzGDAVKqOYRyiFsHYRtmUATBow4CxUmwCGXGg3rN23PmEEjxU9Pcca-iqYaoEOlNq6+oZhjubyQA)